src/screens/Customers/components/CustomerCard.tsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import { createStyles, Theme, withStyles } from '@material-ui/core/styles';
+import { Card, CardActions, Typography, IconButton } from '@material-ui/core';
+import ArrowForwardIosIcon from '@material-ui/icons/ArrowForwardIos';
+import AttachMoneyIcon from '@material-ui/icons/AttachMoney';
+import FlashOnIcon from '@material-ui/icons/FlashOn';
+
+const styles = (theme: Theme) =>
+  createStyles({
+    arrow: {
+      color: theme.palette.text.secondary,
+      fontSize: '15px',
+      marginLeft: '0px',
+    },
+    cardContent: {
+      flexGrow: 1,
+      display: 'flex',
+      alignItems: 'center',
+      paddingLeft: '17px',
+    },
+    totalText: {
+      color: theme.palette.text.secondary,
+    },
+    titleText: {
+      fontSize: '19px',
+    },
+    numberText: {
+      color: theme.palette.error.main,
+      marginLeft: '5px',
+    },
+    divSpacing: {
+      paddingLeft: '15px',
+    },
+    singleCard: {
+      width: '100%',
+      height: '93px',
+      border: '1px solid ' + theme.palette.text.disabled,
+      borderRadius: '8px',
+      textAlign: 'left',
+      display: 'flex',
+      flexDirection: 'row',
+      marginBottom: '10px',
+    },
+    active: {
+      height: '7px',
+      width: '7px',
+      borderRadius: '50%',
+      backgroundColor: theme.palette.secondary.light,
+    },
+    notActive: {
+      height: '7px',
+      width: '7px',
+      borderRadius: '50%',
+      backgroundColor: theme.palette.text.disabled,
+    },
+    arrowSpacing: {
+      padding: '0px',
+    },
+    indicator: {
+      display: 'flex',
+      alignItems: 'center',
+      marginRight: '10px',
+    },
+    icon: {
+      borderRadius: '8px',
+      color: 'white',
+      padding: '5px',
+      backgroundColor: theme.palette.primary.main,
+      fontSize: '30px',
+    },
+  });
+
+interface CardProps {
+  name: string;
+  meterNumber: number;
+  amount: string;
+  active: boolean;
+  status?: CustomerStatus;
+  classes: {
+    arrow: string; cardContent: string; singleCard: string; totalText: string; numberText: string; divSpacing: string; active: string; notActive: string; arrowSpacing: string; indicator: string; icon: string; titleText: string;
+  };
+}
+
+export enum CustomerStatus {
+  METER = 'Meter' as any,
+  PAYMENT = 'Payment' as any,
+  DONE = 'Done' as any,
+}
```

Comment:
It would be better to put this somewhere more accessible. Maybe `customerData.ts` instead because it's not intuitive that this enum should live here